adventurelib.py

```diff
@@ -523,6 +552,17 @@ def start(help=True):
         qmark.orig_pattern = '?'
         commands.insert(0, (Pattern('help'), help, {}))
         commands.insert(0, (qmark, help, {}))
+    try:
+        _commands = []
+        for pattern, func, kwargs in _available_commands():
+            _commands.append(pattern.prefix[0])
+        _commands = list(set(_commands))
+        completer = TabCompleter(_commands)
+        readline.set_completer_delims(' \t\n;')
+        readline.parse_and_bind("tab: complete")
+        readline.set_completer(completer.complete)
```

Comment:
Here, too possible my me because of [this](https://stackoverflow.com/questions/5637124/tab-completion-in-pythons-raw-input).